octobot_evaluators/data/__init__.py

```diff
@@ -13,3 +13,7 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
+
+
+class UnsetTentacleEvaluation(Exception):
```

Comment:
I would prefer, to have an error.py like other OctoBot packages. It's also safer with cython to not use _init_.py